docs/Bitbucket.md

```diff
@@ -0,0 +1,169 @@
+> - `branches`: Defines the branch-specific configuration of a Bitbucket pipeline.
+> - `custom`: A customised starting condition whereby only manual triggering is allowed.
+> - `default`: The default starting configuration of a Bitbucket pipeline if it does not fall into one of the other conditions.
+> - `pull_request`: Defines the pull-request specific configuration of a Bitbucket pipeline.
+> - `tags`: Defines the tag specific configuration of a Bitbucket pipeline.
+> 
+> For information on each of these individual starting conditions, refer to the reference within the Bitbucket Pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/pipeline-start-conditions/#Custom--manual--pipeline-variables).
+
+### Branches (`pipelines.branches`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `pipelines.branches` | ✅ | Application of specific Bitbucket pipeline configuration based for specific branches. Translated to a [step conditional](https://buildkite.com/docs/pipelines/conditionals#conditionals-in-steps) in the corresponding Buildkite pipeline utilising the `build.branch`/`BUILDKITE_BRANCH` variable. |
+| `pipelines.branches.<branch>` | ✅ | The branch name/wildcard to apply specific Bitbucket pipeline step configuration within. |
+| `pipelines.branches.<branch>.parallel` | ✅ | Parallel (concurrent step) configuration for a specific branch within a Bitbucket pipeline. View the available parallel [properties](#parallel-pipelinestypeparallel) supported by the Migrations Tool - and additional property information in the Bitbucket Pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/parallel-step-options/#Parallel). |
+| `pipelines.branches.<branch>.step` | ✅ | Individual step configuration for a specific branch within a Bitbucket pipeline. See configuration options in this section (`pipelines.default.step.<property>`) for supported/unsupported properties. |
+| `pipelines.branches.<branch>.stage` | ✅ | Stage configuration for a specific branch within a Bitbucket pipeline. View the available stage [properties](#stage-pipelinestypestage) supported by the Migrations Tool - and additional property information in the Bitbucket Pipelines [documentation](https://support.atlassian.com/bitbucket-cloud/docs/stage-options/#Stage). |
+
+### Custom (`pipelines.custom`)
+
+| Key | Supported? | Notes |
+| --- | --- | --- |
+| `pipelines.custom` | ✅ | Bitbucket pipelines that are only able to be triggered manually. The corresponding Buildkite pipeline is first generated with an [input step](https://buildkite.com/docs/pipelines/input-step) before any command jobs to ensure that triggered builds must be manually processed. |
+| `pipelines.custom.<name>` | ✅ | The name of the custom Bitbucket pipeline. |
+| `pipelines.custom.<name>.import-pipeline.import` | ❌ | The specification of importing a pipeline from within a specific repository defined in top-level `definitions`. |
```

Comment:
I may be inclined to point them to replacing this with a trigger step instead

Reply:
Added a line-item on trigger steps here